CHANGELOG.md

```diff
@@ -9,7 +9,28 @@ _With the release of `v3.0.0`, we're introducing a new changelog format in an at
 
 _The old changelog can be found in the `release-2.6` branch_
 
-# Changes Since v3.0.1
+# Changes Since v3.0.2
+
+# v3.0.2 - []
+
+  - Added build requirements to RPM spec file for easier RPM building
+  - Replaced the `$SINGULARITY_NOHTTPS` environment variable to allow pulls from http instead of https
+  - Replaced the `$SINGULARITY_TMPDIR` environment variable for builds performed by the build, actions and pull commands
+  - Added the ability to pull from a private shub registry
+  - Replaced the `$SINGULARITY_DOCKER_USERNAME` and `$SINGULARITY_DOCKER_PASSWORD` environment variables for authentication and introduced the `--docker-login` flag for interactive authentication
+  - Made the following packages open:
```

Comment:
This should not be documented in the changelog. This is already documented in the commit log, and is too granular for a changelog